tools/hangar/__snapshots__/test_corpus/valid/debug_env.test.w_test_sim.md

```diff
@@ -2,7 +2,7 @@
 
 ## stdout.log
 ```log
-[symbol environment at ../../../../examples/tests/valid/debug_env.test.w:7:5]
+[symbol environment at debug_env.test.w]
```

Comment:
imo this should either be relative to cwd or absolute.
Either way, it should keep the line:column

Reply:
Fixed so the line:column are shown